lang/cs/Org.Apache.REEF.Wake/Remote/IDataReader.cs

```diff
@@ -143,5 +144,12 @@ public interface IDataReader
         /// <returns>Task handler that reads bytes and returns that number of bytes read 
         /// if success, otherwise -1</returns>
         Task<int> ReadAsync(byte[] buffer, int index, int bytesToRead, CancellationToken token);
+
+        /// <summary>
+        /// Gets underlying stream. Throws null exception if 
+        /// stream is null or not available
+        /// </summary>
+        /// <returns>The underlying stream</returns>
+        Stream GetStream();
```

Comment:
Is this safe if the other methods in this interface are also used? @anupam128 Can you have a look please?